eng/restore/netcoreapp/netcoreapp.depproj

```diff
@@ -4,15 +4,21 @@
   <PropertyGroup>
     <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
     <NETCoreAppPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp2.0'">2.0.7</NETCoreAppPackageVersion>
-    <NETCoreAppPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp2.1'">2.1.3</NETCoreAppPackageVersion>
+    <NETCoreAppPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp2.1'">2.1.3</NETCoreAppPackageVersion>        
+    <NETCoreAppRefPackageVersion Condition="'$(TargetFramework)' == 'netcoreapp3.0'">3.0.0-preview7-27912-14</NETCoreAppRefPackageVersion>
     <BinPlaceRef>true</BinPlaceRef>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != 'netcoreapp3.0'">
     <PackageReference Include="Microsoft.NETCore.App">
       <Version>$(NETCoreAppPackageVersion)</Version>
     </PackageReference>
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
+    <PackageReference Include="Microsoft.NETCore.App.Ref" Version="$(NETCoreAppRefPackageVersion)" ExcludeAssets="All" />
+    <NuGetDeploy Include="$(NuGetPackageRoot)\microsoft.netcore.app.ref\$(NETCoreAppRefPackageVersion)\ref\$(TargetFramework)\*.dll" NuGetPackageId="Microsoft.NETCore.App.Ref" NugetPackageVersion="$(NETCoreAppRefPackageVersion)" />
```

Comment:
I don't think this is the right way to do it. You can do it directly with updating the `KnownFrameworkReference`.

You will probably need to set `<DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>` when `TargetFramework` == `netcoreapp3.0`